lib/pfconfig/namespaces/config/Source.pm

```diff
@@ -0,0 +1,71 @@
+package pfconfig::namespaces::config::Source;
+
+=head1 NAME
+
+pfconfig::namespaces::config::Source
+
+=cut
+
+=head1 DESCRIPTION
+
+pfconfig::namespaces::config::Source
+
+This module creates the configuration hash associated to source.conf
```

Comment:
Is source.conf really a thing now ? I thought it was based off authentication.conf like before